### PR TITLE
Add documentation for storage live migration

### PR DESCRIPTION
**Pull request by awels**
Add documentation on creating an example environment and MigPlan that allows for storage live migration.

### CODE REVIEW

File: docs/scenarios/virtual-machines/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+| Move | No  | |
+
+### Prerequisites
+
+Referring to the getting started [README.md](https://github.com/konveyor/mig-controller/blob/master/README.md), you'll first need to deploy mig-controller and Velero, and then create the following 'Mig' resources on the cluster where mig-controller is running to prepare for Migration:
+
+Also deploy [KubeVirt](https://github.com/kubevirt/kubevirt/) and the Containerized Data Importer [CDI](https://github.com/kubevirt/containerized-data-importer/?tab=readme-ov-file#deploy-it) according to the documentation provided there. This will add the appropriate Custom Resource Definitions (CRD) used by mig-controller to manipulate the Virtual Machines.
+
+**_NOTE:_** If the mig-controller pod was started before KubeVirt and CDI was installed, the mig-controller will not automatically see the CRDs were installed. You have to (re)start the mig-controller pod after installing KubeVirt and CDI.
```
Comment (aglitke) [inline docs/scenarios/virtual-machines/README.md:20]: Can we make this automatic by adding a watch in the mig controller for the kubevirt resources?

Comment (awels) [reply docs/scenarios/virtual-machines/README.md:20]: We can, but it is not a trivial change. Have to watch CRDs. We did something similar in CDI for snapshot CRDs and it was a pain to get right.

File: docs/scenarios/virtual-machines/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Migration toolkit for containers now supports storage migration for KubeVirt virtual machines. Migration can happen offline (virtual machine is turned off) as well as online (virtual machine is running).
+
+Main use cases for this functionality is migrating from one storage class to another. For instance if one is rebalancing across different storage providers, or some new storage is available that is better suited for the workload running inside the VM.
+
+### Supported PV Actions
+
+| Action | Supported | Description |
+|-----------|------------|-------------|
+| Copy | Yes | Create new PV in *same namespace*. Copy data from source PV to target PV, and modify Virtual Machine definition to point to the new PV. If liveMigrate flag is set, the VM will live migrate. If not set, the VM is shutdown, the source PV contents are copied to the target PV and the VM is started again |
```
Comment (duyanyan) [inline docs/scenarios/virtual-machines/README.md:11]: should be PVC instead?

Comment (awels) [reply docs/scenarios/virtual-machines/README.md:11]: No, it should be PVs in this context. The UI refers to these as PVs everywhere so I wanted to be consistent with that in the documentation.

**Pull request merged**